users/views.py

```diff
@@ -1,3 +1,27 @@
-# from django.shortcuts import render
+from rest_framework.authtoken.models import Token
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
-# Create your views here.
+from users.models import User
+from users.serializers import UserPublicSerializer
+
+
+class GetUser(APIView):
+    def get(self, request):
+        users_objects = User.objects.all()
+        serializer = UserPublicSerializer(users_objects, many=True)
+        return Response({"status": 200, "payload": serializer.data})
+
+
+class RegisterUser(APIView):
+    def post(self, request):
+        serializer = UserPublicSerializer(data=request.data)
```

Comment:
How does the user set their password?

Reply:
Serializer and new view for sign up were implemented